### PR TITLE
delete all things

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 include n.Makefile
 
+verify:
+	# verify is not needed - this is all yaml
+
```
Comment (i-like-robots) [inline Makefile]: definitely an opportunity for ascii

Comment (i-like-robots) [inline Makefile]:
```
echo
 _        _______  _ 
( (    /|(  ___  )( )
|  \  ( || (   ) || |
|   \ | || |   | || |
| (\ \) || |   | || |
| | \   || |   | |(_)
| )  \  || (___) | _ 
|/    )_)(_______)(_)
                     
```

**Comment by i-like-robots** [comment]
do it

File: Makefile (hunk 1 above)
Comment (aintgoin2goa) [inline Makefile]: yeah - but I don't know how to make that work in a makefile...

**Pull request merged**